userguide/storage.rst

```diff
@@ -225,27 +230,11 @@ the details when considering whether encryption is right for your
   inaccessible. Always back up the key!
 
 * The encryption key is per ZFS volume (pool). Multiple pools each
-  have their own encryption key.
-
-#ifdef freenas
```

Comment:
This #ifdef is needed.  All TrueNAS hardware has AESNI, and does not need this section.

Reply:
If there's a page you can link which gives the preprocessor directives usable in this doc, it would help - I can see there's an `#ifdef ... #endif`, but is the matching syntax `#ifdef ... #else ... #endif`, or `#ifndef ... #endif` for any alternate wording?

Reply:
In the main directory, run `./preproc -h` for the full help information on the preprocessor.  The #ifdef and #endif terms match: in this case, it would be `#ifdef truenas` and `#endif truenas`.